x-pack/legacy/plugins/apm/public/components/app/Settings/CustomizeUI/CustomActions/EmptyPrompt.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiButton, EuiEmptyPrompt } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+export const EmptyPrompt = ({
+  setIsFlyoutOpen
```

Comment:
I don't think a state updater function should leak into this component. Preferably this component has a `onCreateCustomActionClick` prop (or something similarly named), and leave it up to the consuming component to decide how to handle the event.